### DISCUSSION TITLE
Where does the processing occur?

### DISCUSSION THREAD

**Discussion by Noah-Jaffe** [linto-ai/whisper-timestamped#206] [General]
If I wanted to use this to transcribe mp4 files, would this send ANY data off of the localhost? Is there any processing done by externally controlled entities? 
(Knewbie here lost in the sauce) thank you

**Comment by Jeronymous** [comment]
whisper-timestamped just operates a model that is local. All data is processed on the local CPU (GPU if any). No data is sent anywhere (unless done externally to whisper-timestamped)
